lms/djangoapps/instructor/tests/test_api.py

```diff
@@ -136,6 +139,407 @@ def test_alreadyrunningerror_ajax(self):
         self.assertIn("Task is already running", result["error"])
 
 
+@override_settings(MODULESTORE=TEST_DATA_MIXED_MODULESTORE)
+class TestEmailQueries(ModuleStoreTestCase, LoginEnrollmentTestCase):
+    """
+    Ensures the backend logic is sound for instructor email widget
+    """
+
+    OPEN = "opened"
+    COMPLETED = "completed"
+    NOT_OPEN = "not opened"
+    NOT_COMPLETED = "not completed"
+
+    def setUp(self):
+        self.course = CourseFactory.create()
+        course_id = 'edX/emailToy/2015_Winter'
+        self.course_key = SlashSeparatedCourseKey.from_deprecated_string(course_id)
+        self.instructor = InstructorFactory(course_key=self.course_key)
+        self.client.login(username=self.instructor.username, password='test')
+        self.open_user = UserFactory()
+        self.completed_user = UserFactory()
+        self.not_opened_user = UserFactory()
+        self.inactive_user = UserFactory()
+        self.problem = self.course_key.make_usage_key('problem', '123')
+        StudentModule.objects.create(student=self.open_user,
+                                     course_id=self.course_key,
+                                     module_state_key=self.problem,
+                                     )
+        StudentModule.objects.create(student=self.completed_user,
+                                     course_id=self.course_key,
+                                     module_state_key=self.problem,
+                                     grade=5,
+                                     )
+        output = [  # pylint: disable=unused-variable
```

Comment:
Why are we assigning this to a variable if we're not using the value?

Couldn't we just do:
```python
for user in [...]:
    CourseEnrollment.objects.create(...)
```